react_juce/core/ViewManager.cpp

```diff
@@ -235,6 +271,27 @@ namespace reactjuce
         throw std::logic_error("Caller attempted to invoke method on non-existent View instance");
     }
 
+    juce::var ViewManager::createStyleSheet(const juce::var& value)
+    {
+        jassert(value.isObject());
+
+        juce::DynamicObject::Ptr res = new juce::DynamicObject();
+
+        auto obj = value.getDynamicObject();
+
+        for (auto &nv : obj->getProperties())
+        {
+            jassert(nv.value.isObject());
+
+            const StyleSheetID id = juce::DefaultHashFunctions::generateHash(juce::Uuid(), INT_MAX);
```

Comment:
What if we hashed the actual property set as the ID? That way if a user accidentally (or intentionally) built two style sheet objects that were identical to one another, our map would coalesce them and save some space

Reply:
Yeah doesn't seem a bad idea. Moderate concerns around collisions and performance after some recent findings. Could do in this PR or as enhancement/performance follow up ?